docusaurus/docs/welcome/instructions/app-icon.md

```diff
@@ -37,6 +37,9 @@ To start the Asset Studio, follow these steps:
 The app icon should have both a square and a round version.
 There tool offers a few options which you can tweak as much as you like, but we will only cover the necessary ones.
 
+> ℹ️  The following instructions won't cover how to generate [adaptive icons](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive), which are icons that can display a variety of shapes across different device models.
```

Comment:
I still need to add add some information about from which Android version the adaptative icons are fully supported and may be necessary, since some OEMs may need them due to the way they manipulate the icons